tests/cli/output/mini_d_ucwd.txt

```diff
@@ -1624,6 +1624,15 @@ nl_between_get_set              = 0        # unsigned number
 # Add or remove newline between C# property and the '{'.
 nl_property_brace               = ignore   # ignore/add/remove/force
 
+# The number of newlines after '{' of a namespace. This also adds newlines
+# before the matching '}'.
+# 
```

Comment:
O frabjous day! Our first trailing whitespace error (which apparently github doesn't show) in these files. Apparently none of our descriptions have blank lines yet. (In #1852, quite a few will have blank lines... Note to self: check that #1852 is clever enough to omit the trailing space.)